### PR TITLE
update compatibility table

### LINKED ISSUE #2231: Update incompatibility table
### URL, if applicable

https://docs.quantum.ibm.com/guides/runtime-options-overview#options-compatibility

### Describe the new content you are requesting.

Fractional gates and gen3-turbo should be added to the compatibility table. 

Also, should it really be called `Options compatibility`?  Things like dynamic circuits and pulse gates are not really options. 

### If this new content request is accepted, do you want to write the content?

I can help the team by providing enough information to write the material

### PR DESCRIPTION
**Pull request by beckykd**
Closes #2231

### CODE REVIEW

**Comment by beckykd** [comment]
Changed topic: https://qiskit.github.io/documentation/pr-2392/guides/runtime-options-overview#feature-compatibility

File: docs/guides/runtime-options-overview.mdx [hunk 1]
```diff
@@ -177,9 +189,20 @@ Due to differences in the device compilation process, certain runtime features c
   Can be used with pulse gates, but it might not work when using custom gates.
   </TabItem>
 
+  <TabItem value="Gen3" label="gen3-turbo">
```
Comment (jyu00) [inline docs/guides/runtime-options-overview.mdx:192]: There is actually another `gen3-turbo` tab already all the way in the end

File: docs/guides/runtime-options-overview.mdx [hunk 2]
```diff
@@ -177,9 +189,20 @@ Due to differences in the device compilation process, certain runtime features c
...
+  Incompatible with:
+  - Dynamic circuits
+  - Fractional gates
+  - Pulse gates
+  - `meas_type=kerneled` or `meas_type=avg_kerneled` (measurement level 1)
+
+  Does not support parameter expressions with arithmetic operations.
```
Comment (jyu00) [inline docs/guides/runtime-options-overview.mdx:199]: Why is this not a bullet point?

Comment (beckykd) [reply docs/guides/runtime-options-overview.mdx:199]: I guess those extra notes usually have a 'but', so I can pull that parameter expressions one into the bullets.

File: docs/guides/runtime-options-overview.mdx (hunk 1 above)
Comment (beckykd) [reply docs/guides/runtime-options-overview.mdx:192]: yep, I just noticed that and deleted it so they're alphabetical.

**Review by jyu00 — APPROVED** [review]
Thanks @beckykd!

**Review by abbycross — APPROVED** [review]

**Pull request merged**